pyOCD/tools/flash_tool.py

```diff
@@ -147,6 +147,8 @@ def main():
     else:
         board_selected = MbedBoard.chooseBoard(board_id=args.board_id, target_override=args.target_override,
```

Comment:
You are right, I accidentally removed this change in my previous update. The new version should restore the initial parameter setting.